solr/solrj/src/java/org/apache/solr/common/util/JavaBinCodec.java

```diff
@@ -24,6 +24,10 @@
 import java.lang.invoke.MethodHandles;
 import java.nio.ByteBuffer;
 import java.nio.file.Path;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
```

Comment:
Did you mean to include LocalDateTime in the else/if below?

Reply:
I did until I noticed that I don't have enough information to convert LocalDateTime. Thanks for spotting. I've removed the import.